input_data.go

```diff
@@ -6,7 +6,9 @@ type Modifier int
 
 var (
```

Comment:
Chose to export `mouse` after renaming it to `MouseState`. (since it holds information about the current state of the mouse). 

If we ever want to move it to `engo.Input`, then that's for another PR.